From ba14494f02966f16afb7ffe098afe339ada783df Mon Sep 17 00:00:00 2001
From: Jiang, Bin <bin.jiang@windriver.com>
Date: Mon, 18 Apr 2011 12:00:22 +0800
Subject: [PATCH 11/13] Fix: high resolution timer failed to start

The high resolution timer failed to start, because clocksource CSRC_R4K
wasn't registered.

Signed-off-by: Jiang Bin<bin.jiang@windriver.com>
---
 arch/mips/wintegra/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/mips/wintegra/Kconfig b/arch/mips/wintegra/Kconfig
index cfe11ec..c8840e9 100644
--- a/arch/mips/wintegra/Kconfig
+++ b/arch/mips/wintegra/Kconfig
@@ -7,6 +7,7 @@ config WINTEGRA_WINPATH3
         bool "Support for Wintegra Winpath3 device family"
         select BOOT_ELF32
         select CEVT_R4K
+        select CSRC_R4K
         select IRQ_CPU
         select SYS_HAS_CPU_MIPS32_R2
         select DMA_COHERENT
-- 
1.7.0.4

